scripts/cluster_k3d.sh

```diff
@@ -8,10 +8,10 @@ set -aeEuo pipefail
 K3S=${K3S:-$(k3d version -o json | jq -r '.k3s')}
 CLUSTER_NAME=${CLUSTER_NAME:-k3s-default}
 MASTER_COUNT=${MASTER_COUNT:-1}
-WORKER_COUNT=${WORKER_COUNT:-1}
+WORKER_COUNT=${WORKER_COUNT:-0}
```

Comment:
Can we have 0 workers? Is this means that we will use only server nodes in the testing cluster? If that works, I have nothing against it. I'm just curious about that.

Reply:
Agents are useful to test deamon sets, taints, scheduling, ...
Otherwise it works the same way, 0 workers is default [in k3d](https://k3d.io/stable/usage/commands/?h=agents).

Originally I added one worker to simulate "real world", I am not sure if it's swill worth it.